websites/N/Nexon Logistics/presence.ts

```diff
@@ -0,0 +1,174 @@
+const presence = new Presence({
+		clientId: "1046638491435552819",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", async () => {
+	let presenceData: PresenceData = {
+		largeImageKey: "logo",
+	};
+	const [buttons, timestamp, privacy] = await Promise.all([
+			presence.getSetting<boolean>("buttons"),
+			presence.getSetting<boolean>("timestamp"),
+			presence.getSetting<boolean>("privacy"),
+		]),
+		pages: Record<string, PresenceData> = {
+			"/about": { details: "Viewing About Page" },
+			"/gallery": { details: "Viewing Gallery Page" },
+			"/events": { details: "Browsing Events Page" },
+			"/newsletter": { details: "Browsing Newsletter" },
+			"/rules": { details: "Reading the rules" },
+			"/PrivacyPolicy": { details: "Reading Privacy Policy" },
+			"/Terms&Condition": { details: "Reading Terms&Condition" },
+			"/leaderboard": { details: "Viewing Leaderboard" },
+		};
+
+	for (const [path, data] of Object.entries(pages)) {
+		if (document.location.pathname.includes(path))
```

Comment:
Properties of `document.location` are used many times in this presence. Destructure these properties for consistency/simplicity.

Reply:
fixed where it was possible

Reply:
This was not fixed.